app/src/main/res/layout/activity_parking_stay.xml

```diff
@@ -0,0 +1,176 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/background_parking_stay"
+    tools:context=".activity.ParkingStayActivity">
+
+    <TextView
+        style="@style/text_view_style"
+        android:id="@+id/start_date_text_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/date_start_text"
+        android:textAppearance="@style/TextAppearance.AppCompat.Body2"
+        app:layout_constraintBottom_toTopOf="@id/end_date_text_view"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toStartOf="@id/start_time_selected_text_view"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        style="@style/text_view_style"
+        android:id="@+id/start_date_selected_text_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:ems="5"
+        app:layout_constraintBottom_toTopOf="@id/start_time_selected_text_view"
+        app:layout_constraintStart_toEndOf="@id/start_date_text_view"
+        app:layout_constraintEnd_toStartOf="@id/start_date_time_button"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        style="@style/text_view_style"
+        android:id="@+id/start_time_selected_text_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:ems="5"
+        app:layout_constraintBottom_toTopOf="@id/end_date_selected_text_view"
+        app:layout_constraintStart_toEndOf="@id/start_date_text_view"
+        app:layout_constraintEnd_toStartOf="@id/start_date_time_button"
+        app:layout_constraintTop_toBottomOf="@id/start_date_selected_text_view" />
+
+    <TextView
+        style="@style/text_view_style"
+        android:id="@+id/end_date_text_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/date_end_text"
+        android:textAppearance="@style/TextAppearance.AppCompat.Body2"
```

Comment:
this property its common for others text view, here you need to use a different style

Reply:
done